core/spec.md

```diff
@@ -5387,7 +5426,8 @@ When serialized as a JSON object, the Version entity adheres to this form:
 
   "RESOURCEurl": "URL", ?                  # If not local
   "RESOURCE": ... Resource document ..., ? # If inlined & JSON
-  "RESOURCEbase64": "STRING" ?             # If inlined & ~JSON
+  "RESOURCEbase64": "STRING", ?            # If inlined & ~JSON
+  "ancestor": "STRING"
```

Comment:
let's move this into the previous section since that's where all of the "normal" attributes appear.
I like it between "modifiedat" and "contenttype" since it's kind of related to timestamps.

Reply:
I think there are other pseudo schema chunks that need to be touched by this PR to add the attribute - eg the main one at the top of the doc.

Reply:
I hope I got them all, please take a look